src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -439,7 +440,7 @@ const useGroupInfo = () => {
         data: {
           user_id: profile.myProfile.user_id,
           other_id: item?.user_id || item?.userId,
-          username: item?.user?.name || item?.user?.username
+          username: item?.user?.name || item?.user?.username || item.username
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This line of code is quite complex and hard to read due to the multiple nullish coalescing operators. Consider breaking it down into simpler steps or adding comments to clarify what each part does.
```diff
- username: item?.user?.name || item?.user?.username || item.username
+ // Extract the username from the item object
+ const userName = item?.user?.name || item?.user?.username || item.username;
+ username: userName
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->